receiver/sapmreceiver/trace_receiver.go

```diff
@@ -208,10 +210,15 @@ func New(
 	if err != nil {
 		return nil, fmt.Errorf("failed to marshal default response body for %v receiver: %w", config.ID(), err)
 	}
+	transport := "http"
```

Comment:
This can be inlined in the New call

Reply:
What do you mean by this can be inlined in the New call?